google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GoogleDataprocInterpreter.scala

```diff
@@ -280,13 +280,16 @@ private[google2] class GoogleDataprocInterpreter[F[_]: Parallel](
 
       // Add back the preemptible instances, if any
       _ <- numPreemptibles match {
-        case Some(n) =>
+        case Some(n) if n != 0 =>
```

Comment:
@kyuksel this is where we're checking there's indeed preemptible

Reply:
Would it be safer as follows?

```suggestion
        case Some(n) if n > 0 =>
```